src/test/it/essentials.yml

```diff
@@ -0,0 +1,19 @@
+---
+ath:
+  useLocalSnapshots: false
```

Comment:
It should not be necessary to specify this explicitly.

Reply:
This concern has been raised by me during the review of https://github.com/jenkins-infra/pipeline-library/pull/35 . Decision by @raul-arabaolaza was to keep the default behavior as `true`. It is documented here: https://github.com/jenkins-infra/pipeline-library/blob/master/vars/runPCT.txt#L87